smart-contracts/contracts/BridgeProxy/ArbitrumOrbitNativeBridgeProxy.sol

```diff
@@ -35,7 +35,8 @@ contract ArbitrumOrbitNativeBridgeProxy is BridgeProxy {
     address asset, //l2 token
     address l1Currency, //l1 token
     uint256 amount,
-    bytes calldata /* data*/
+    bytes calldata /* data */,
```

Comment:
maybe we could find a abetter name than `data` here ? like `txParams` since it contains `abi.encode(nonce, recipient, amount, block.timestamp)` ?

Reply:
its a bit redundant with `extraData`

Reply:
yeah, good point, I will update!